CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# Contributing to this repo
+
+## Overview
+There are two components to the [dbt Hub](https://hub.getdbt.com/):
+
+- **hub.getdbt.com:** Our package registry
+    - https://github.com/dbt-labs/hub.getdbt.com (this repo!)
+    - An indexed collection of JSON files
+    - A static site, served by those same JSON files, where users can discover available packages
+    - Hosted on Netlify
+- **Hubcap:** The way the Hub stays up to date!
+    - https://github.com/dbt-labs/hubcap
+    - A script that:
+        - Catches new package releases when they are available (via GitHub releases)
+        - Parses out metadata—crucially, this package's dependencies on *other* Hub packages
+        - Records each release in a predictable format (JSON file)
+        - Automatically opens a PR against [hub.getdbt.com](http://hub.getdbt.com) to add that JSON file to its indexed collection
+    - Hosted on Heroku
+
+## Running locally
+
+### Development
+
+See the [installing Ruby for Mac](#installing-ruby-for-mac) and [troubleshooting](#troubleshooting) sections below as-needed.
+
+Clone this repository and install submodules:
+
+```
+git submodule update --init
+```
+
+...then install dependencies with `bundler`:
+
+```bash
+bundle install
+```
+
+...and run with middleman:
+
+```bash
+bundle exec middleman serve --port 4567
+```
+
+### Use with `dbt deps` from dbt Core
+
+You can set the `DBT_PACKAGE_HUB_URL` environment variable to specify where `dbt deps` should look to resolve dependencies [listed in `packages.yml`](https://docs.getdbt.com/docs/building-a-dbt-project/package-management#how-do-i-add-a-package-to-my-project):
+```shell
+export DBT_PACKAGE_HUB_URL=http://localhost:4567/
```

Comment:
Perhaps instructions on how to delete this afterwards as well? Or at least a reminder _to_ delete it afterwards

Reply:
Very good point! Committed an update to incorporate this feedback.